gpt_engineer/core/default/steps.py

```diff
@@ -0,0 +1,257 @@
+from gpt_engineer.core.code import Code
+from gpt_engineer.core.ai import AI
+from gpt_engineer.core.chat_to_files import parse_chat
+from gpt_engineer.core.default.paths import ENTRYPOINT_FILE, CODE_GEN_LOG_FILE
+from gpt_engineer.data.file_repository import FileRepository
+from langchain.schema import HumanMessage, SystemMessage
+
+import inspect
+import os
+import re
+from termcolor import colored
+import subprocess
+
+#TODO: THIS NEEDS A BETTER SOLUTION
+PREPROMPTS_PATH = os.path.join("gpt_engineer", "preprompts")
```

Comment:
Shouldnt pre prompts just be injected as constant strings from a python module

Reply:
Just want to point out the history here, with the ambition that it should be easy to switch out the preprompts by the _user_ of gpt-engineer.

(Or even use something like scale.ai spellbook in the future)

I'm all for simplifying now and add that back later.